src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonService.kt

```diff
@@ -35,7 +35,8 @@ class PrisonService(
         prisonNumber = it.offenderNo,
         date = it.movementDate,
         pncNumber = null,
-        isCurrentPrisoner = true
+        isCurrentPrisoner = true,
+        gender = null
```

Comment:
Left prison transfers as null, doesn't seem like we're getting any gender data from here, do we want to enrich this with BaSM data as well? Or is it completely separate?

Reply:
yep, keep it separate.